### ISSUE TITLE
On example number 2 in demo, it allows adding a tag that is already present

### ISSUE THREAD

**Issue by PTC-JoshuaMatthews** [Gbuomprisco/ngx-chips#820]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[ X] bug report => search github for a similar issue or PR before submitting
[ ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**
If using objects, it allows duplicating a tag that is already present.

**Expected behavior**
Should see the tag is a duplicate and follow the configured duplicate handling settings 

**Minimal reproduction of the problem with instructions (if applicable)**
Go to demo page and attempt to add "React" to the second demo

**What do you use to build your app?. Please specify the version**
Angular CLI

**Angular version:** 
6.0.3

**ngx-chips version:**
1.9.7 and tested on 1.9.4

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
All browsers

**Comment by boyardnicolas** [comment]
I have the same problem and I thought I was doing something wrong, actually it is more subtle than that.
You cannot add a duplicate of new tags : take an existing tag on the demo and try adding it twice, first try will work, not the second one. Seems there is a difference between a tag that has just been added and one that has been retrieved from the model upon initialization.

**Comment by Gbuomprisco** [comment]
This is due to the fact it checks for the value of the ID rather than the display value. It's a difficult one as different people think it should work in different ways. What I could do is to provide an option for a custom diff.

**Comment by boyardnicolas** [comment]
Hello Giancarlo,
Why checking the ID? The display value is the entirety of the information of a tag and a user wouldn't be able to differentiate two different tags with the same name. Actually here it doesn't really check the ID since the ID of new tags is filled with the display value.

**Comment by PTC-JoshuaMatthews** [comment]
@Gbuomprisco  I agree with @boyardnicolas . A key aspect of a tag control is that all tags can be distinguished from one another, we should assume display value is the unique identifier for purposes of duplicate detection.  Are there really people that think the control should allow duplicate tags without enabling that option?

**Comment by PTC-JoshuaMatthews** [comment]
@Gbuomprisco  Any update on this? Do you plan to change the behavior or should I refactor away from using objects model?
